content/docs/5.1/develop/clients/java.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Java Client
+description: Interact with TiKV using Java.
+menu:
+    "5.1":
+        parent: TiKV Clients
+        weight: 1
+---
+
+This guide introduces how to interact with TiKV using [Java Client](https://github.com/tikv/client-java).
+
+{{< warning >}}
+TiKV Java Client is developed and released using Java8. The minimum supported version of TiKV is 2.0.0.
+{{< /warning >}}
+
+## Add the dependency
+
+To start, open the `pom.xml` of your project, and add the `tikv-client-java` as dependencies if you are using Maven.
+
+```xml
+<dependency>
+  <groupId>org.tikv</groupId>
+  <artifactId>tikv-client-java</artifactId>
+  <version>${latest_version}</version>
```

Comment:
should we replace ${latest_version} with the later version number?

Reply:
If we use the latest version number here, it must be updated each time a new version is released.

Reply:
Another problem, the example code here uses the API in version `3.1.0-SNAPSHOT` which is not released yet.

Reply:
update to `3.2.0-SNAPSHOT`